packages/medusa/src/strategies/batch-jobs/product/import.ts

```diff
@@ -463,6 +486,42 @@ class ProductImportStrategy extends AbstractBatchJobStrategy {
     }
   }
 
+  private async assignProductToCollection(
+    product: Product,
+    collectionData: { title: string; handle: string }
```

Comment:
comment: We don't use title for anything, so we should we change the method signature to be `assignProductToCollection(product, collectionHandle)`?

Reply:
I ve changed the implementation, this method does not exists anymore. I am testing before pushing